tough/tests/http.rs

```diff
@@ -96,7 +96,20 @@ mod http_integ {
     pub fn integ_dir() -> PathBuf {
         let mut p = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
         p.pop();
-        p.join("integ").canonicalize().unwrap()
+        p = p.join("integ");
+        p
+    }
+
+    /// Returns a command object that runs the provided script under BASH , wether we are under cygwin or unix.
+    pub fn bash_base() -> Command {
+        // if under cygwin, run the bash script under cygwin64 bash
+        if cfg!(target_os = "windows") {
+            let mut command = Command::new("c:\\cygwin64\\bin\\bash");
```

Comment:
```
            let mut command = Command::new("c:\\cygwin64\\bin\\bash");
```

The absolute path here causes me concern. It may be the most likely place for cygwin bash to be installed but it could actually be anywhere. Is it typical to not have cygwin's bin dir in the machine's PATH?

Reply:
I looked a bit into it, but it doesn't seem like they are any way to retrieve from the cmd shell the cygwin64 home path :(, when I looked up integration with cygwin from other apps, it looked like hard-coding the cygwin root path wasn't a that bad thing.

Reply:
,and `cygpath` isn't available from `cmd` or `powershell`, so ... :(

Reply:
https://github.com/awslabs/tough/pull/342#discussion_r576441363
they *aren't

Reply:
This make me sad as well. Did you try running this tests from cygwin ? I think bash command will be available there directly. If tests works on cygwin we can document to run this tests only from cygwin or related unix style terminal for windows because of shell script dependency. 
Alternative could be to re-write `run.sh` as powershell script if that is possible 😅

Reply:
Okay, I agree with you. What is the error message if `cygwin` does not exist at path `c:\\cygwin64\\bin\\bash` ? If it already does not say, can we have a way to inform user that `cygwin` is expected to be located at `c:\\cygwin64` or something similar. It's well documented in Readme, may be an overkill but seems helpful according to me, if error message is misleading and actual reason was cygwin not found at expected location.